www/src/pages/components/modal-layer.mdx

```diff
@@ -18,13 +18,13 @@ A generic component for creating accessibile modal layer objects. The modal laye
 
 ```jsx live
 () => {
-  const [isOpen, open, close] = useToggle(false);
+  const [isOpen, open, onClose] = useToggle(false);
```

Comment:
Would prefer to keep these names not mirror onClose (setClose or close)

Reply:
kk, will change back!